### DISCUSSION TITLE
SalesForce Soql Join Script

### DISCUSSION THREAD

**Discussion by forzamami** [dbeaver/dbeaver#18793] [Q&A]
Hi ,
I just started using this batch. I can connect to Salesforce, I see tables and data, but I could not join. How can I join please help?

Thank You

**Comment by LonwoLonwo** [comment]
```sql
SELECT
Id,
OCE__Account__r.Id
from
OCE__AccountTerritoryFields__c,
OCE__Account__r
```
?
